### ISSUE TITLE
Move Repos/Collections nav and clarify landmark roles 

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#624]
Goal is to move the Repositories and Collections links to the masthead section, to allow for expansion of the search feature (#555 TBD) and clarification of landmarks.

- [ ] Move the links to the masthead `<div>`, to the far right side of the `<h1>` (see screenshots below)
   - on small devices, wrap and center below the `<h1>`
- [ ] Add `aria-label="utilities"` to `<nav class="navbar navbar-expand-md navbar-dark bg-dark topbar" role="navigation">` 
- [ ] Add `aria-label="browse"` and `role="navigation"` to the Repos/Collections `<ul>` you just moved
- [ ] Remove `role="navigation"` from the `<div>` that now only contains Search.

**Before**
<img width="1151" alt="move browse before" src="https://user-images.githubusercontent.com/6945691/63624417-1abd2f00-c5b1-11e9-9ea2-59b4dd1fa751.png">

**After**
<img width="1168" alt="move browse after" src="https://user-images.githubusercontent.com/6945691/63624420-1db81f80-c5b1-11e9-9350-1450775cef1e.png">

**After mobile**
<img width="396" alt="Screen Shot 2019-08-23 at 1 06 56 PM" src="https://user-images.githubusercontent.com/6945691/63624449-345e7680-c5b1-11e9-82b2-855026ce0ba9.png">

**Comment by camillevilla** [comment]
Closed by #662
